### PR TITLE
Initial implementation of UI E2E test

### PR DESCRIPTION
**Pull request by Comoque1**
### Describe the background of your pull request

Simple Protractor E2E UI test.
To run the test first should be applied `npm install`, password for the user in `conf.js` and run the command: `protractor conf.js` (from the file location)

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [x] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE 
     and the contributor license agreement: tba

### CODE REVIEW

File: conf.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// conf.js
+exports.config = {
+  framework: 'jasmine',
+  directConnect: true, // enabled, when used without webdriver-manager
+  seleniumAddress: 'http://localhost:4444/wd/hub',
+  specs: ['test/**/*.spec.js'],
+  capabilities: {
+    browserName: 'firefox',
+    'moz:firefoxOptions': {
+      args: ['--safe-mode'],
```
Comment (valentinyanakiev) [inline conf.js:10]: why safe mode?

File: conf.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    },
+  },
+
+  params: {
+    login: {
+      url:
+        'https://login.microsoftonline.com/22e3aada-5a09-4e2b-9e0e-dc4f02328b29/oauth2/v2.0/authorize?client_id=6f4fcdb2-5792-4e17-bbdc-14ac0cf78496&scope=openid%20profile%20offline_access&redirect_uri=https%3A%2F%2Fdev.cherrytwist.org%2Flogin&client-request-id=ad17f83e-e49c-4a06-be8c-c59f800b2410&response_mode=fragment&response_type=code&x-client-SKU=msal.js.browser&x-client-VER=2.9.0&x-client-OS=&x-client-CPU=&client_info=1&code_challenge=xSV2NcP3-qE0J0-rzPqBVgMOatMMbVg6I2wFQ-q0Bbo&code_challenge_method=S256&nonce=0c5eabab-69f7-4899-a5ba-b3e8ed744188&state=eyJpZCI6IjliZDA3NjFhLTlkY2EtNGI5Yi1hM2MxLTIzMzdlMDYxNTFjNSIsInRzIjoxNjEzNTE2ODc3LCJtZXRhIjp7ImludGVyYWN0aW9uVHlwZSI6InBvcHVwIn19',
```
Comment (valentinyanakiev) [inline conf.js:17]: is the state url param always the same?

**Review by valentinyanakiev — COMMENTED** [review]
Overall seems ok. A bit monolithic, but can be improved upon in the future.

File: conf.js (hunk 1 above)
Comment (Comoque1) [reply conf.js:10]: This is an example given by protractor.
From what I saw,  `--safe-mode` reduce the tests running inconsistently and increase the execution speed.

File: conf.js (hunk 2 above)
Comment (Comoque1) [reply conf.js:17]: Yes.

**Review by atstojanov — COMMENTED** [review]
More reliable way of selecting elements is required.
Here is example recommended priority list of selecting elements:
https://testing-library.com/docs/queries/about 
The latest fallback should be `data-testid` attribute, which we don't have in the code currently, but can be added. The good thing is that this attribute can be omitted in the production code. (more research needed)

**Pull request merged**